pkg/ui/react-app/src/thanos/pages/blocks/BlockDetails.tsx

```diff
@@ -80,6 +82,39 @@ export const BlockDetails: FC<BlockDetailsProps & PathPrefixProps> = ({
             <b>Chunks:</b> <span>{block.stats.numChunks}</span>
           </div>
           <hr />
+          {sizeStats && (
+            <>
+              <div data-testid="total-size">
+                <b>Total size:</b>&nbsp;
```

Comment:
Those non breaking spaces are mostly because Prettier (?) trims my trailing spaces at the end of the line :(